examples/date-picker.py

```diff
@@ -0,0 +1,113 @@
+import dash
+from dash import dcc, html
+from dash.dependencies import Input, Output, State
+import plotly.express as px
+import pandas as pd
+import numpy as np
+
+app = dash.Dash(__name__)
+
+app.layout = html.Div(
+    [
+        dcc.Store(id="stored-data", storage_type="memory"),
+        html.H1("Date Picker Example"),
+        html.H2("Step 1: Select a Date Range to generate ramdom data"),
+        html.P("Start Date"),
+        dcc.DatePickerSingle(
+            id="date-picker-single-start", clearable=True, display_format="YYYY-MM-DD"
```

Comment:
In order to make the ids unique across all the files in this project, we use a naming convention of `<filename>`-x- as a prefix.  So for example, this id should be `"date-picker-x-single-start"`

All the ids in this app need to be updated.  For more information please [Contributing Guilde](https://github.com/AnnMarieW/dash-app-gallery/blob/main/CONTRIBUTING.md)